ico/tests/contracts/test_forwarder.py

```diff
@@ -105,7 +110,7 @@ def test_halt(web3, payment_forwarder, team_multisig, customer):
     team_multisig_begin = web3.eth.getBalance(team_multisig)
     payment_forwarder.transact({"from": team_multisig}).halt()
     with pytest.raises(TransactionFailed):
-        payment_forwarder.transact({"value": value, "from": customer}).pay(customer_id, customer)
+        payment_forwarder.transact({"value": value, "from": customer}).payWithoutChecksum(customer_id, customer)
 
 
 def test_unhalt(web3, payment_forwarder, team_multisig, customer):
```

Comment:
Can you also add a reverse test that we check if Invested event comes through with a correct checksum and we can verify this in Python land?

Reply:
Is this request satisfied by the recent commit?